docker/client.py

```diff
@@ -737,7 +738,7 @@ def start(self, container, binds=None, port_bindings=None, lxc_conf=None,
               dns=None, dns_search=None, volumes_from=None, network_mode=None,
               restart_policy=None, cap_add=None, cap_drop=None, devices=None,
               extra_hosts=None, read_only=None, pid_mode=None, ipc_mode=None,
-              security_opt=None, ulimits=None):
+              security_opt=None, ulimits=None, cgroup_parent=None):
```

Comment:
Gladly! You can see the [documentation](http://docker-py.readthedocs.org/en/latest/hostconfig/) on that topic.